common/crypto_utils/Makefile

```diff
@@ -38,10 +38,13 @@ install:
 	pip3 install $(WHEEL_FILE)
 
 clean:
-	if [ -f $(WHEEL_FILE) ] ; then pip3 uninstall --yes $(WHEEL_FILE); fi
 	rm -rf build deps dist *.egg-info
-	rm -f crypto/crypto.py crypto/crypto_wrap.cpp
-	rm -f verify_report/verify_report.py verify_report/verify_report_wrap.cpp
+	rm -f avalon_crypto_utils/crypto/crypto.py
+	rm -f avalon_crypto_utils/crypto/crypto_wrap.cpp
+	rm -f avalon_crypto_utils/verify_report/verify_report.py
+	rm -f avalon_crypto_utils/verify_report/verify_report_wrap.cpp
+	if [ -f $(WHEEL_FILE) ] ; then pip3 uninstall --yes $(WHEEL_FILE); fi
```

Comment:
Generally wheel file will be copied to deps folder in the directory which has Makefile. We are already deleting build folder before this. Then why do we need this if check?

Reply:
Ya Manju , all wheel file is getting copied to dist folder not deps. So will remove the step of exclusively deleting the wheel file.

Reply:
done